client/lib/yourturn/src/app.jsx

```diff
@@ -1,23 +1,41 @@
 import React from 'react';
 import Sidebar from './sidebar/sidebar.jsx';
 import NotificationBar from './notification-bar/notification-bar.jsx';
+import DeprecationWarning from '../../common/src/components/DeprecationWarning.jsx'
 
-const YourTurn = (props) => {
+class YourTurn extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {show: true};
+        this.dismiss = this.dismiss.bind(this);
+    }
+
+    dismiss() {
+        this.setState({show:false});
+    }
+
+    render() {
     return (<div className='yourturn'>
                 <NotificationBar />
-                <div className='grid with-gutter'>
+        <div className='grid with-gutter'>
                     <div className='grid-col col-1-4'>
                         <Sidebar
-                            activeRoute={props.location.pathname} />
+                            activeRoute={this.props.location.pathname} />
                     </div>
                     <div className='grid-col'>
+                        {this.props.location.pathname === '/' ?
+                            <DeprecationWarning dismissable={false} /> : null
```

Comment:
Can you please remove all the code, that handles the dismissing stuff? Sorry for the extra work, but it doesn't fell right to merge unused code.